packages/chatgpt/src/Adaptor.js

```diff
@@ -0,0 +1,61 @@
+import {
+  execute as commonExecute,
+  composeNextState,
+} from '@openfn/language-common';
+import { expandReferences } from '@openfn/language-common/util';
+import { request } from './Utils';
+import 'dotenv/config';
```

Comment:
We use `state.configuration` for setting up credentials, please remove this library